package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "validatore-xml-190",
+  "version": "0.0.1",
+  "description": "",
+  "main": "src/server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/synapta/validatore-XML-190.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
there used to be a GPLv2 license in the repo that got removed by mistake i guess @alemela?

Reply:
se mi dite la riaggiungo in questa PR

Reply:
@alemela ↑